core/domain/feedback_services.py

```diff
@@ -406,16 +437,67 @@ def clear_feedback_message_references(user_id, exploration_id, thread_id):
         model.put()
 
 
+def get_all_recipient_list(exploration_id, thread_id, author_id):
+    """Returns dict containing batch recipients and other recipients
+    for sending feedback message email.
+    """
+    exploration_rights = rights_manager.get_exploration_rights(exploration_id)
+
+    owner_list = set(exploration_rights.owner_ids)
+    participant_list = get_all_thread_participants(exploration_id, thread_id)
+    sender = set([author_id])
+
+    batch_recipients = owner_list - sender
+    other_recipients = participant_list - batch_recipients - sender
+
+    return {
+        'batch_recipients': batch_recipients,
```

Comment:
I was initially going to do the same. I thought that returning arbitrary number of values would lead to confusion and that is why I used dict instead. If you think that it is fine to use it without dict then no problem.